subjects/tetrisoptimizer.audit.en.md

```diff
@@ -0,0 +1,62 @@
+#### Functional Project Questions
+
+##### Try bad example 00 (github.com/public/subject/badexample00.md).
```

Comment:
Please move the example files in a folder dedicated to the exercise.
For example for tetrisoptimizer : public/subjects/tetrisoptimizer/examples/*.md